unit/cq_test.c

```diff
@@ -172,21 +172,14 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((op = getopt(argc, argv, "f:a:h")) != -1) {
+	while ((op = getopt(argc, argv, INFO_OPTS)) != -1) {
```

Comment:
We would want to limit this to just "f:p:h" since this test doesn't use -e option.